ui/app/css/index.scss

```diff
@@ -22,5 +22,9 @@
 @import './itcss/objects/index';
 @import './itcss/components/index';
 @import './itcss/trumps/index';
-// Other imports
+
+/*
+  Third Party Library Styles
```

Comment:
nit:
```suggestion
 * Third Party Library Styles
```

Reply:
There should be a lint rule for this.

Reply:
There is a lint rule for this!

Reply:
But it doesn't do what I want 😢

Reply:
The comment style matches the other comment styles in this file and other scss files, at least some of which are pre-existing. I agree we should have an enforced comment style in scss files, though I'm curious if that has been decided and whether or not I can apply that to this file when I update it to remove all of the itcss imports?

Reply:
gotta find a style-lint plugin for it ;)

Reply:
TIL it's a good idea to consider the file extension when raving about styles 🤦‍♂️